libs/flowbite-angular/src/lib/components/modal/modal.component.ts

```diff
@@ -62,6 +78,17 @@ export class ModalComponent extends BaseComponent {
     createClass({ modalContainerClass: '', modalContentClass: '', rootClass: '' }),
   );
 
+  //#region template properties
+  @ViewChild('modal', { read: TemplateRef })
```

Comment:
Instead of `@ViewChild`, please use [`viewChild`](https://angular.dev/api/core/viewChild?tab=api) signal based initializer.
You will be able to delete `// @ts-expect-error: Property 'template' has no initializer` comment